src/System.Web.HttpUtility/tests/HttpUtility/HttpUtilityTest.cs

```diff
@@ -412,6 +412,7 @@ public void UrlDecode(string decoded, string encoded)
 
         [Theory]
         [MemberData(nameof(UrlDecodeData))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Full framework assumes the current codepage is UTF8")]
```

Comment:
I'm not understanding the need for this.  Are all of the inputs problematic?  Just one of the inputs?  The test is explicitly specifying UTF8.